c/feature-dictionary.asciidoc

```diff
@@ -2,6 +2,14 @@
 // Creative Commons Attribution 4.0 International License; see
 // http://creativecommons.org/licenses/by/4.0/
 
+// “generic” opencl_c_feature_name
+ifdef::backend-html5[]
+:opencl_c_feature_name: pass:q[`\__opencl_c_<wbr>&lt;feature_<wbr>name&gt;</em>`]
+endif::[]
+ifndef::backend-html5[]
+:opencl_c_feature_name: pass:q[`\__opencl_c_&#8203;<feature_&#8203;name>`]
```

Comment:
I'm seeing the follow error on lines where this substitutes for PDF generation.

> asciidoctor: ERROR: failed to parse formatted text:

Side-note: Travis actually saw this error too, but `asciidoctor` still has a `0` exitcode and produces a PDF.  So the build passes.

Copying the HTML approach more closely seems to work:

```suggestion
:opencl_c_feature_name: pass:q[`\__opencl_c_&#8203;&lt;feature_&#8203;name&gt;`]
```

Reply:
Oops, sorry. I can't build the PDF documentation so I missed it, and Travis not complaining didn't help 8-). I've pushed a rebased branch with the wrapper macro fixed (hopefully!)

Reply:
Thanks, no more build problems now.